packages/neuron-ui/src/states/stateProvider/actionCreators/wallets.ts

```diff
@@ -113,7 +113,8 @@ export const sendTransaction = (params: Controller.SendTransaction) => (dispatch
       if (res.status) {
         history.push(Routes.History)
       } else {
-        addNotification({ type: 'alert', content: JSON.stringify(res.message.title) })(dispatch)
+        // TODO: the pre-treat is unnecessary once the error code is implemented
```

Comment:
What is the error code change is expected to finish? If that's very soon we can combine these two changes together.

Reply:
The CKB team is going to add error codes in the document, but the deadline is not determined.

Reply:
I see.